Cargo.toml

```diff
@@ -53,9 +54,11 @@ tiny_http = { version = "0.12", optional = true }
 ed25519-dalek = "2"
 once_cell = "1"
 p256 = { version = "0.13", features = ["ecdsa"] }
+x509-cert = { version = "0.2.4", features = ["builder"] }
 
 [features]
 default = ["http", "passwords", "setup"]
+der-signer = ["spki", "sha2?/oid"]
```

Comment:
I think you can go ahead and make this an always-on non-feature since the requisite dependencies are already required